lib/charms/layer/apache_zeppelin.py

```diff
@@ -168,3 +170,22 @@ def close_ports(self):
     def cleanup(self):
         self.dist_config.remove_dirs()
         unitdata.kv().set('zeppelin.installed', False)
+
+
+class ZeppelinAPI(object):
+    """
+    Helper for interacting with the Appache Zeppelin REST API.
+    """
+    def _url(self, *parts):
+        url = 'http://localhost:9090/api/'
```

Comment:
Let's use the `DistConfig().port('zeppelin')` port instead of 9090:

https://github.com/juju-solutions/layer-apache-zeppelin/blob/master/dist.yaml#L32